src/web3/constants.ts

```diff
@@ -17,6 +17,28 @@ export const NETWORK_NAMES = {
   1337: 'Local network',
 };
 
+interface Network {
```

Comment:
This isn't a descriptive name for this type, and it's also not very type safe.

I also just realised we will have a save wrapper per-mAsset.

Perhaps better:

```
interface Addresses {
  MTA: string;

  mUSD: {
    SaveWrapper: string;
  };
}

type AddressesByNetwork = Record<typeof CHAIN_ID, Addresses>;

const ADDRESSES_BY_NETWORK: AddressesByNetwork // ...

export const ADDRESSES: Addresses = ADDRESSES_BY_NETWORK[CHAIN_ID];
```